crowd-server/crowd_server/urls.py

```diff
@@ -21,5 +21,6 @@
     path('admin/', admin.site.urls),
     path('image-label/', include('crowd_server.apps.image_label.urls')),
```

Comment:
You can use`path('image-label/', include('image_label.urls')),`, because django can discover your app from `settings.py`